ocs_ci/ocs/resources/pvc.py

```diff
@@ -120,3 +123,33 @@ def get_all_pvcs(namespace=None):
     )
     out = ocp_pvc_obj.get()
     return out
+
+
+def verify_pv_exists_in_backend(
+    pv_name, pool_name, namespace=defaults.ROOK_CLUSTER_NAMESPACE
+):
+    """
+    Verifies given pv exists in ceph backend
+
+    Args:
+        pv_name (str): Name of the pv
+        pool_name (str): Name of the rbd-pool
+        namespace (str): Name of namespace
+    Returns:
+         bool: True if pv exists on backend, False otherwise
+    """
+    spec_volhandle = "'{.spec.csi.volumeHandle}'"
+    cmd = f"oc get pv/{pv_name} -o jsonpath={spec_volhandle} -n {namespace}"
+    out = run_cmd(cmd=cmd)
+    image_uuid = "-"
+    image_uuid = image_uuid.join(out.split('-')[5:10])
+    cmd = f"rbd info -p {pool_name} csi-vol-{image_uuid}"
+    ct_pod = pod.get_ceph_tools_pod()
+    try:
+        ct_pod.exec_ceph_cmd(
+            ceph_cmd=cmd, format='json'
+        )
+    except CommandFailed as ecf:
+        log.error(f"PV is not found on ceph backend: str{ecf}")
+        return False
```

Comment:
There are chances that 'CommandFailed' occurs due to some other reason. Non existence of PV should be verified using the error message.

Reply:
I am expecting 3 outcome here. 1. PV is present, 2. PV is absent, 3. Failed to fetch the required details.
Return value is True for case 1, False for case 2.
@RazTamir Please advice how to handle these 3 cases without using assertion. Or is it fine to use assert for case 3?

Reply:
> There are chances that 'CommandFailed' occur due to some other reason. Non existence of PV should be verified using the error message.

Even for non-existence of PV fails with 'CommandFailed' and returns False.